### PR TITLE
contribution guidelines

### PR DESCRIPTION
**Pull request by StevenLangbroek**
## Contribution guidelines

I could definitely use some help on this, but what I have so far is the [Contributor Covenant](https://www.contributor-covenant.org/).

### Todo
- [x] Add code of conduct
- [x] Contribution guidelines sketch

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
Comment (jxnblk) [inline CONTRIBUTING.md]: 👍

**Comment by jxnblk** [comment]
Sorry to leave this hanging, I sorta think the `CONTRIBUTING.md`file should include some of the things you mention in the description and that the CoC should be renamed to `CODE_OF_CONDUCT.md` and linked to from the Contributing doc (this seems a little more standard IMO). If you're down to make those changes, I can go ahead and merge what's here and we can follow up with some of the other parts in other PRs. What do you think?

**Comment by StevenLangbroek** [comment]
Sure thing!

**Comment by StevenLangbroek** [comment]
Not sure what's going on with Netlify, but I've made the changes you requested. Apologies for taking so long.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (StevenLangbroek) [inline CODE_OF_CONDUCT.md]: @jxnblk do you want your e-mail address here or a link to GH issues for example?

Comment (jxnblk) [reply CODE_OF_CONDUCT.md]: You can add `jxnblk@gmail.com` here – I think it's important for people to be able to report any potential issues privately

Comment (StevenLangbroek) [reply CODE_OF_CONDUCT.md]: Good call 🙇

**Comment by jxnblk** [comment]
@StevenLangbroek oh, and you can ignore the netlify stuff. That should settle down once another PR is merged in, but doesn't reflect anything wrong with this branch

**Comment by StevenLangbroek** [comment]
Alright ✌️, anything else I can do to help (in general)?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Contribution Guidelines
+
+WIP
+
```
Comment (jxnblk) [inline CONTRIBUTING.md]: Maybe should go ahead and add a link to the CoC here?

**Comment by StevenLangbroek** [comment]
@jxnblk added a couple of lines about getting started :v:

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+We follow the Contributor Covenant. You can read it, and where to report abuse in our [Code of Conduct](./CODE_OF_CONDUCT.md).
+
+## How to Contribute
+
+If you'd like to fix a bug, add a feature, or just generally tinker with
+`styled-system`, please start by forking the repository, and creating a new
+branch. Then install dependencies by running `npm install` (we use `npm`, you're
+free to use `yarn`, just take care to not commit `yarn.lock`).
+
+You can start the documentation development server by executing `npm start`. By
```
Comment (jxnblk) [inline CONTRIBUTING.md]: I just ripped all this out, but don't worry, I can update this docs after merging 😬

**Comment by jxnblk** [comment]
Thanks for all your help with this! Going to merge this in and make a few edits to the contributing doc since I changed all the docs setup. Also heads up, that I'll be moving this over to the `styled-system` org after this so be sure to keep an eye out for that

**Pull request merged**